chainer/functions/connection/n_step_lstm.py

```diff
@@ -255,50 +349,50 @@ def n_step_lstm_base(
     Args:
         n_layers(int): Number of layers.
         dropout_ratio(float): Dropout ratio.
-        hx (chainer.Variable): Variable holding stacked hidden states.
+        hx (~chainer.Variable): Variable holding stacked hidden states.
             Its shape is ``(S, B, N)`` where ``S`` is number of layers and is
             equal to ``n_layers``, ``B`` is mini-batch size, and ``N`` is
-            dimention of hidden units.
-        cx (chainer.Variable): Variable holding stacked cell states.
+            dimension of hidden units.
+        cx (~chainer.Variable): Variable holding stacked cell states.
             It has the same shape as ``hx``.
-        ws (list of list of chainer.Variable): Weight matrices. ``ws[i]``
-            represents weights for i-th layer.
+        ws (list of list of :class:`~chainer.Variable`): Weight matrices.
+            ``ws[i]`` represents weights for i-th layer.
```

Comment:
... represents the weights of the i-th layer

Reply:
I fixed next commit.